rfcs/rfc-0002/README.md

```diff
@@ -0,0 +1,143 @@
+
+## Communication
+How will this group organize itself and its work (which GitHub repos, projects, communication channels, etc.)? 
+
+How will progress be reviewed and reported to the TSC or other stakeholders? Include the frequency of reviews and the format of progress reports.
+```
+
+### Drawbacks
+
+**Overhead**: Working Groups / chartering could be seen as heavy handed. An alternative could be to have a set of work items approved by the TSC and not form working groups at all. This could lead to more overhead, slowing down decision-making and execution.
+
+Counterpoint: Having a distinct set of responsibilities and deliverables can be a powerful driver of focused delivery and accountability.
+
+**Flexibility**: Charters may limit the flexibility of Working Groups to pivot or adapt to new information or changing circumstances. A more flexible approach could allow for more dynamic and responsive project management.
+
+Counterpoint: Having a straightforward process for charter amendment could limit this risk.
+
+**Complexity in Coordination**: Managing multiple Working Groups with their own charters could create complexity in coordination, especially if there are interdependencies between groups. A more centralized approach might simplify coordination and reduce potential conflicts.
+
+Counterpoint: This trades off giving more autonomy and flexibility to those closest to the work.
+
+#### Alternative Approaches:
+
+**Ad Hoc Teams**: Instead of formal Working Groups, form ad hoc teams for specific tasks or projects. These teams can be more flexible and nimble, adapting quickly to changes without the need for formal charters.
+
+**Task Forces**: Establish task forces for urgent or high-priority initiatives. Task forces can have a narrow focus and short lifespan, providing a more agile approach to addressing specific issues.
+
+**Centralized Management**: Centralize the management of work items under the TSC. This could streamline processes and reduce the overhead associated with multiple Working Groups.
+
+## Prior Art
+
+The proposed charter template was heavily influenced by the [W3C Charter Templateh](https://w3c.github.io/charter-drafts/charter-template.html) as well as the [DIF Charter Documents](https://github.com/decentralized-identity/org/tree/master/Org%20documents/WG%20documents).
+
+## Unresolved Questions
+
+1. How to operate a WG (tooling & meeting/comms norms).
+
+A GitHub repo could be created per WG; though it could also be viable to track WGs via the TSC repository.
+
+Similarly meetings/comms norms could be handled by each group, with common suggestions for async processes (like using GitHub issues and lableing to track work).
+
+2. Handling participation.
+
+Should we track who is participating in each working group? Or stick to a top level DRI for the Working Group and each of its deliverables? How do we handle changes over time?
```

Comment:
per the meeting on 5/30, we're leaning on sticking a top level DRI for the working group who is the tiebreaker, if consensus cannot be reached